tests/fixtures_process_monitor.py

```diff
@@ -12,6 +12,7 @@ def fixture_test_monitor():
     def _foo(process_manager):
         svd = process_manager.values_to_share_to_server
         svd["system_status"] = SERVER_INITIALIZING_STATE
+        svd["fe_be_connection_success"] = True
```

Comment:
This fixture sets up the bare minimum in the shared_values_dict only, which is `system_status = SERVER_INITIALIZING_STATE`, so this key should not be set here